cea/interfaces/dashboard/api/project.py

```diff
@@ -298,3 +311,7 @@ def get(self, scenario):
         else:
             abort(400, 'Scenario does not exist', choices=choices)
 
+
+def get_scenarios_from_project(config):
```

Comment:
This could be moved to either a method on config (`scenarios`) or a module-level function in `cea.config` - though... first check if it's used anywhere else in the code: If not, then we just leave it here...

Reply:
should probably be called "list_scenario_names_for_project" (as we tend to use the name "scenario" to refer to the whole path most of the time.